src/main/java/seedu/sudohr/model/ModelManager.java

```diff
@@ -91,6 +98,16 @@ public void setSudoHrFilePath(Path sudoHrFilePath) {
         userPrefs.setSudoHrFilePath(sudoHrFilePath);
     }
 
+    @Override
+    public void refresh() {
```

Comment:
Just double confirm, setting predicate here to be true / false of an already filtered list will not introduce new unnecessary behaviour right? it wont remove all entires or display all entries right

Reply:
It tested abit an it has no produced any unnecessary behaviour for now